intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -1516,6 +1561,33 @@ def convert_date_utc(value: str) -> Optional[str]:
     },
 }
 
+accessible_rdp_eudp = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port'),
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'hostname'),
+        ('classification.identifier', 'tag'),  
+        ('source.asn', 'asn', invalidate_zero),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'sessionid', validate_to_none),
+        
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'vulnerable',
+        'classification.type': 'vulnerable-system',
+        'classification.identifier': 'open-rdp',
```

Comment:
```suggestion
        'classification.identifier': 'open-rdpeudp',
```
The difference is important as RDP alone is non susceptible to DDoS, but RDPEUDP is.